frontend/src/components/molecules/NextStepButton/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { EnrollmentStatus } from '../../../config/status-parameters';
+import Button from '../../atoms/hyperTexts/Button';
+import { useDataProviderConfigurations } from '../../templates/hooks/use-data-provider-configurations';
+
+const NextStepButton = ({
+  target_api,
+  status,
+}: {
+  target_api: string;
+  status: EnrollmentStatus;
+}) => {
+  const { dataProviderConfigurations } = useDataProviderConfigurations();
```

Comment:
You can use the useDataProvider hook here to avoid using too much optional chaining operator.